src/Docnet.Tests.Integration/PageRangeValidatorTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using Docnet.Core.Validation;
+using Docnet.Tests.Integration.Utils;
+using Xunit;
+
+namespace Docnet.Tests.Integration
+{
+    public class PageRangeValidatorTests : IClassFixture<PageRangeValidatorFixture>
```

Comment:
It is not like `PageRangeValidator` is a stateful class, we do not need this complexity with `IClassFixture`, just create `Sut` in a transient fashion for each test.

Reply:
Good catch. Removed fixture.